### PR TITLE
Update authentication.md

### PR DESCRIPTION
**Pull request by ansysic**
Removed unused request parameters. These are not needed to complete the POST request. I wondered why they were there in the first place.

### CODE REVIEW

**Comment by timdorr** [comment]
These are required query parameters for the POST request. These cannot be removed and still have the request complete.

**Comment by ansysic** [comment]
Sorry, perharps could you shed some light on the post request mentioned above. My code example works as followed:
**Step 1: Obtain the login page**
Outputs HTML including hidden forms details. `curl -G` is a GET request
```
curl -G --cookie-jar cookies.txt \
    --data-urlencode "client_id=ownerapi" \
    --data-urlencode "code_challenge=OWJlNWRhMjBhNGIxZTM2ZWI5ZDM3NzUzM2JhNjA3MTUzMjgwMWY2OWQ5NmEzN2YzYmJiMTlkMDk3OTVhZjM0MSAgLQo=" \
    --data-urlencode "code_challenge_method=S256" \
    --data-urlencode "redirect_uri=https://auth.tesla.com/void/callback" \
    --data-urlencode "response_type=code" \
    --data-urlencode "scope=openid email offline_access" \
    --data-urlencode "state=123123123" \
    --data-urlencode "login_hint=email@tesla.com" \
    https://auth.tesla.com/oauth2/v3/authorize

```
**Step 2: Obtain an authorization code**
Outputs a redirect location via header.
Now this is where i get confused with your documentation. From above obtained hidden forms I create a POST request. Why do we need GET parameters from Step 1 again? It also states in the documentation itself, that the following paramters are not part of the post body:

> Note: These are query parameters, not part of the POST body

```
curl -s --cookie cookies.txt -D header.txt \
    --data-raw "_csrf=PRGhaD6v-V33fG-FZmmEq66fN6sF7lcwQ6ss" \
    --data-raw "_phase=authenticate" \
    --data-raw "_process=1" \
    --data-raw "transaction_id=GeWvF4AT" \
    --data-raw "cancel=" \
    --data-raw "identity=email@tesla.com" \
    --data-raw "credential=password" \
    https://auth.tesla.com/oauth2/v3/authorize
```

### To summarize
Why not remove the second listing of GET request parameters in your documentation, if they are not needed in the POST request? They are needed in Step 1, but not in Step 2.

What am I missing?

Best regards, 
Phil ;)
